### ISSUE TITLE
Disabling analytics also disables userAnalyticsEvent hook

### ISSUE THREAD

**Issue by aoumiri** [onfido/onfido-sdk-ui#986]
Thanks to https://github.com/onfido/onfido-sdk-ui/issues/884, we are now able to disable Onfido's own analytics, Woopra. Which is great.

But I have a suggestion of improvement regarding the [hook override feature](https://github.com/onfido/onfido-sdk-ui#overriding-the-hook).

### What was the expected behaviour?

By using the `disableAnalytics`, I would have expected only the analytics going to Woopra to be disabled, and not the `userAnalyticsEvent` hook.

Indeed, those events can be used for example for:

- custom analytics, other than Woopra.
- use cases other analytics. It's really great to know in which step the user is right now, and react to it (for an evenn deeper UX integration, for example).

### What happened instead?

All the events sent through the `userAnalyticsEvent` are also disabled.

### Version info:

- Onfido SDK: 5.9.0
- Browser: FF 75

### Integration configuration:

```js
    onfido.init({
      token,
      containerId,
      language: {
        locale: opts.locale,
        phrases: opts.phrases,
        mobilePhrases: opts.mobilePhrases,
      },
      userDetails: { smsNumber: opts.smsNumber },
      steps: [
        'welcome',
        'document',
        {
          type: 'face',
          options: {
            requestedVariant: 'video',
            uploadFallback: true,
          },
        },
        'complete',
      ],
      smsNumberCountryCode: opts.smsNumberCountryCode || 'FR',
      onComplete: opts.onComplete,
      onError: opts.onError,
      disableAnalytics: true,
    });
```

and the code that handles the hook:

```js
window.addEventListener('userAnalyticsEvent', event => {
  console.log(event.detail.eventName);
});
```

### Steps to reproduce:

Navigate on all steps of the onfido flow, no events when `disableAnalytics: true`.

Thanks for your help !

**Comment by stefaniacardenas** [comment]
Hi @aoumiri, thank you for the suggestion. It seems like a valid point. I will discuss it with the team to evaluate adding this improvement.

**Comment by stefaniacardenas** [comment]
Hi @aoumiri, this has been fixed in version [5.13.0](https://github.com/onfido/onfido-sdk-ui/releases/tag/5.13.0).

**Comment by aoumiri** [comment]
Hey @stefaniacardenas! Thanks a lot for this release. We'll have a look at Qonto as soon as we can.

Have a great day.
